layout/header/header-user/component.js

```diff
@@ -15,18 +17,13 @@ class HeaderUser extends PureComponent {
     setDropdownOpened: PropTypes.func.isRequired
   }
 
-  logout(e) {
-    if (e) e.preventDefault();
-
-    // TO-DO: move this to an action
-    fetch(`${process.env.CONTROL_TOWER_URL}/auth/logout`, { credentials: 'include' })
-      .then(() => {
-        window.location.href = `/logout?callbackUrl=${window.location.href}`;
-      })
-      .catch((err) => {
-        toastr.error('Error', err);
+  onLogout = () => {
+    logout()
+      .catch(({ response }) => {
```

Comment:
Not sure 100% but I don't think you receive here the response. Based on you function you will receive an error object with a `message` property. Could you double check this, please?